src/modules/page-load-errors.js

```diff
@@ -21,10 +21,10 @@
 		return {
 			pages: {
 				failedtoparse: function (key, data) {
-					
+					defaultLoadFatalError(key, data);
```

Comment:
Why not simply use failedtoparse: defaultLoadFatalError instead of wraping it into a function since the params signature is the same ? (this point is valid for line 27 and 30 too)